WooCommerce/WooCommerceTests/ViewRelated/Search/Customer/CustomerSearchUICommandTests.swift

```diff
@@ -0,0 +1,46 @@
+import XCTest
+@testable import WooCommerce
+import Yosemite
+
+final class CustomerSearchUICommandTests: XCTestCase {
+    private let sampleSiteID: Int64 = 123
+
+    override func setUp() {
+        super.setUp()
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func test_searchResultsPredicate_includes_siteID_and_keyword_when_keyword() {
+        // Given
+        let command = CustomerSearchUICommand(siteID: sampleSiteID) { _ in }
+
+        // When
+        let predicate = command.searchResultsPredicate(keyword: "some")
+        let expectedQuery = "siteID == 123 AND ANY searchResults.keyword == \"some\""
```

Comment:
Double check if this actually filters down multiple sites.

Reply:
Confirmed 👍 , for 2 customers with the same name and customer ID, in two different sites, only one is shown.